app/Models/Author.php

```diff
@@ -39,7 +39,7 @@ public function getAbbrAttribute(): string
             return mb_strtoupper(mb_substr($nameArray[0], 0, 1)) . mb_strtoupper(mb_substr($nameArray[1], 0, 1));
         }
 
-        return mb_strtoupper($this->title, 0, 2);
```

Comment:
mb_strtoupper() expects at most 2 parameters, 3 given

Reply:
I believe the original intention was instead of using `mb_strtoupper` rather use `mb_substr` here, not sure though.